models/org_team.go

```diff
@@ -289,6 +312,10 @@ func (t *Team) RemoveRepository(repoID int64) error {
 		return nil
 	}
 
+	if t.IncludesAllRepositories {
```

Comment:
Fixed. It will not be normal to try to remove a repository in this case so enough to have it in the lower function.

Reply:
team.removeRepository is used when a repository is deleted, so that is a valid case to remove it from the team. I added a comment to the function instead.